src/e2e/resources/simple-temple-expected/auth/auth.go

```diff
@@ -48,8 +48,8 @@ type loginAuthResponse struct {
 	AccessToken string
 }
 
-// router generates a router for this service
-func (env *env) router() *mux.Router {
+// defaultRouter generates a router for this service
+func defaultRouter(env *env) *mux.Router {
```

Comment:
When would a router not be a default router?

Reply:
The router gets given to the user for modification, this just defaults to everything we've generated